js/glpi_install.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+let request_running = false;
+
+function startDatabaseInstall(message_element_id, success_html)
+{
+    const message_element = document.getElementById(message_element_id);
+    if (!message_element) {
+        const alert = document.createElement('p');
+        alert.innerText = 'Could not load HtmlElement to append messages to.';
+        document.body.appendChild(alert);
+        throw new Error(alert.innerText);
+    }
```

Comment:
User will be blocked with a feedback message that will probably not help. If you want to handle this case that should not exist, you can create a new `message_element` , append it to the DOM, and continue the process.

Reply:
Solved